moveit_planners/pilz_industrial_motion_planner/include/pilz_industrial_motion_planner/pilz_industrial_motion_planner.h

```diff
@@ -134,6 +134,11 @@ class CommandPlanner : public planning_interface::PlannerManager
   pilz_industrial_motion_planner::JointLimitsContainer aggregated_limit_active_joints_;
 
   /// cartesian limit
+  pilz_industrial_motion_planner::CartesianLimit cartesian_limit_;
```

Comment:
I think this line is causing build failures, and seems we're not using it. Try removing it.